bio/bwameth/memx/wrapper.py

```
@@ -0,0 +1,151 @@
+mem2_idx = {
+    ".c2t",
+    ".c2t.0123",
+    ".c2t.amb",
+    ".c2t.ann",
+    ".c2t.bwt.2bit.64",
+    ".c2t.pac",
+}
+
+# Extract index prefix, and list of index extensions
+index = os.path.commonprefix(snakemake.input.idx)
+index_extensions = set(idx[len(index) - 4 :] for idx in snakemake.input.idx)
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)